### PR TITLE
Rename "list" and "delete" to "containers" and "rm"; add "images" and "rmi" commands

### PR DESCRIPTION
**Pull request by nalind**
Add some minimal image management commands.  Neither is really in scope for a building tool, but it reduces the number of use cases where we need additional tools.

### CODE REVIEW

File: cmd/buildah/main.go [hunk 1]
```diff
@@ -68,6 +68,14 @@ func main() {
 	}
 	app.Commands = []cli.Command{
 		{
+			Name:        "images",
```
Comment (TomSweeneyRedHat) [inline cmd/buildah/main.go]: Thought this was going to be list?

File: cmd/buildah/images.go [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+)
+
+var (
+	imagesFlags = []cli.Flag{
+		cli.BoolFlag{
+			Name:  "quiet",
+			Usage: "omit column headings",
+		},
+		cli.BoolFlag{
+			Name:  "qq",
```
Comment (TomSweeneyRedHat) [inline cmd/buildah/images.go]: Depending on the online context, qq means 'crying eyes' or 'just quit the game as you've no skills'.  Perhaps --onlyIDs  or --short or --min?

File: cmd/buildah/main.go (hunk 1 above)
Comment (nalind) [inline cmd/buildah/main.go]: We're already using "list" for listing containers that buildah is working with, though I could be swayed to "list-images" to parallel with "delete-image".

File: cmd/buildah/images.go (hunk 2 above)
Comment (nalind) [inline cmd/buildah/images.go]: Oh.  I was going for something along the lines of 'docker images -q', but am not opposed to renaming the flag.  Any preference?

Comment (TomSweeneyRedHat) [inline cmd/buildah/images.go]: At first I thought just --q would be good, but then remembered docker images uses --quiet or -q to only show the ids.  Can I vote to change the flags to --omit for the column headings and --quiet for the image ids only?

File: cmd/buildah/main.go (hunk 1 above)
Comment (TomSweeneyRedHat) [inline cmd/buildah/main.go]: My bad, ignore this.  I'd forgotten list was for containers.  Images is good.

File: cmd/buildah/images.go (hunk 2 above)
Comment (nalind) [inline cmd/buildah/images.go]: The --quiet flag is what we already for headings in "list", that'll need to be changed there to keep them consistent.  Updating.

**Comment by rhatdan** [comment]
I would prefer not to add additional verbs, just use buildah list and buildah delete.

**Comment by TomSweeneyRedHat** [comment]
LGTM.  For images docker uses 'images' for list and 'rmi' to remove.  For containers 'ps' and 'rm'.  Should we emulate that or do you think the code should recognize the object and do the right thing?

**Comment by nalind** [comment]
We can overload delete so that it looks for a container first, but with free-form names around, we're inevitably going to need some way to specify what type of thing was intended.  We have distinctly different things to display for images and containers, and every output format I can come up with that encompasses both just looks messier if you only care about one or the other.

**Comment by rhatdan** [comment]
Can't we just force user to specify --image or --container when their is conflict.

**Comment by rhatdan** [comment]
@rh-atomic-bot r+  	0e702d6

**Pull request closed without merging**